### PR TITLE
Preserve STDERR

### PR DESCRIPTION
**Pull request by sminot**
I ran into an error which was very hard to debug because I couldn't see the contents of STDERR.

This should preserve STDERR in the logs without altering the set of files which are written out.

## PR checklist

- [ x ] This comment contains a description of changes (with reason).
- [ ] If you've fixed a bug or added code that should be tested, add tests!
- [ ] If you've added a new tool - have you followed the pipeline conventions in the [contribution docs](https://github.com/nf-core/sarek/tree/master/.github/CONTRIBUTING.md)
- [ ] If necessary, also make a PR on the nf-core/sarek _branch_ on the [nf-core/test-datasets](https://github.com/nf-core/test-datasets) repository.
- [ ] Make sure your code lints (`nf-core lint`).
- [ ] Ensure the test suite passes (`nextflow run . -profile test,docker --outdir <OUTDIR>`).
- [ ] Usage Documentation in `docs/usage.md` is updated.
- [ ] Output Documentation in `docs/output.md` is updated.
- [ ] `CHANGELOG.md` is updated.
- [ ] `README.md` is updated (including new tool citations and authors/contributors).

### CODE REVIEW

**Comment by maxulysse** [comment]
cf https://github.com/nf-core/modules/pull/4520

**Review by maxulysse — APPROVED** [review]

File: .github/workflows/ci.yml [hunk 1]
```diff
@@ -205,6 +205,7 @@ jobs:
           - tags: "cat/cat"
           - tags: "cat/fastq"
           - tags: "custom/dumpsoftwareversions"
+          - tags: "dragmap/align"
```
Comment (FriederikeHanssen) [inline .github/workflows/ci.yml:208]: 😱 why?

File: modules/nf-core/dragmap/align/dragmap-align.diff [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+     tuple val(meta) , path(reads)
+
+--- modules/nf-core/dragmap/align/environment.yml
++++ modules/nf-core/dragmap/align/environment.yml
+@@ -4,7 +4,7 @@
+   - bioconda
+   - defaults
+ dependencies:
+-  - dragmap=1.3.0
++  - dragmap=1.2.1
```
Comment (FriederikeHanssen) [inline modules/nf-core/dragmap/align/dragmap-align.diff:23]: I think we don't need to patch anymore and it should work now with the latest version. The rebuild conda recipe fixed it.

Comment (maxulysse) [reply modules/nf-core/dragmap/align/dragmap-align.diff:23]: ok, I'll do that in a separate PR then, as this is out of scope here

File: .github/workflows/ci.yml (hunk 1 above)
Comment (maxulysse) [reply .github/workflows/ci.yml:208]: https://github.com/nf-core/sarek/issues/1349

**Pull request merged**